src/components/Settings/Subscribe/SubscriptionInfo.js

```diff
@@ -114,11 +118,18 @@ const subscriptionInfo = ({
           fullWidth={false}
           color="primary"
           onClick={() => {
-            window.CdvPurchase.store.manageSubscriptions();
+            isAndroid()
+              ? window.CdvPurchase.store.manageSubscriptions()
+              : onCancelSubscription(ownedProduct);
           }}
           style={{ marginLeft: '1em' }}
         >
-          <FormattedMessage {...messages.manageSubscription} />
+          {' '}
+          {isAndroid() ? (
```

Comment:
We need to add a platform validation. If the purchase was made from Paypal, this button does not have effect on Android  and viceversa. A message can be added to, something like _You made the purchase via PayPal, please manage it from the web_

Reply:
this is now implemented